content/applications/productivity/iot/config/updating_iot.rst

```diff
@@ -0,0 +1,150 @@
+==============
+Updating (IoT)
+==============
+
+Due to the complexity of the :abbr:`IoT (Internet of Things)` box and virtual Windows :abbr:`IoT
+(Internet of Things)` box , the term 'updating' can mean several different things. The actual
+drivers can be updated, the core code on the :abbr:`IoT (Internet of Things)` box can be updated
+and, lastly a new image can be flashed (physical :abbr:`IoT (Internet of Things)` box). This
+document will explore the various ways to update the :abbr:`IoT (Internet of Things)` box to ensure
+smooth operation of :abbr:`IoT (Internet of Things)` box processes and devices.
+
+Handlers (drivers) update
+=========================
+
+There may be some instances where the drivers or interfaces need to be updated for individual
+devices (e.g. scale, measurement tools, etc.). The IoT handlers (drivers and interfaces) code can be
+modified by syncing them with the configured server handler's code. This can be helpful in instances
+where :abbr:`IoT (Internet of Things)` devices (e.g. scales, measurement tools, etc.) are not
+working properly with the :abbr:`IoT (Internet of Things)` box.
+
+For both the Windows :abbr:`IoT (Internet of Things)` (Odoo 16 and higher) and physical :abbr:`IoT
+(Internet of Things)` box, this process can be performed manually from the :abbr:`IoT (Internet of
+Things)` box homepage. Go to the :abbr:`IoT (Internet of Things)` box homepage by navigating to
+:menuselection:`IoT app --> IoT Boxes` and clicking on the :guilabel:`IP address` of the :abbr:`IoT
+(Internet of Things)` box.
+
+Next, click :guilabel:`Handlers list` and then select :guilabel:`Load Handlers` at the bottom of the
+page.
+
+.. image:: updating_iot/load-handlers.png
+   :align: center
+   :alt: Handlers list on an IoT box with the load handlers button highlighted.
+
+.. important::
+   Handlers code is fetched from the configured server and it needs to be up-to-date to have the
+   latest fixes and patches. See :ref:`upgrade/homepage_upgrade`.
```

Comment:
I don't really understand why you put `See :ref:upgrade/homepage_upgrade.`. They are two separate things which update different part of IoT odoo's code.